config/locales/en.yml

```diff
@@ -323,6 +323,8 @@ en:
         new_powertip: These products are available to be added to your inventory. Click 'Add' to add a product to your inventory, or 'Hide' to hide it from view. You can always change your mind later!
 
     orders:
+      capture: "Capture"
+      ship: "Ship"
       bulk_management:
```

Comment:
While we're at it, how do you feel about scoping these a little bit more?
`admin.orders.index.ship` and `admin.orders.index.capture`?